### PR TITLE
feat: adds auto-generated README.md to python sdk services

### PR DESCRIPTION
**Pull request by MelvinKl**

### CODE REVIEW

File: templates/python/README.mustache [hunk 1]
```diff
@@ -3,58 +3,25 @@
...
-- Build date: {{generatedDate}}
-{{/hideGenerationTimestamp}}
-- Generator version: {{generatorVersion}}
-- Build package: {{generatorClass}}
 {{#infoUrl}}
 For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
 {{/infoUrl}}
 
-## Requirements.
+This package is part of the STACKIT Python SDK. For additional information please visit the [github repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```
Comment (joaopalet) [inline templates/python/README.mustache:10]:
```suggestion
This package is part of the STACKIT Python SDK. For additional information, please visit the [GitHub repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```

Comment (joaopalet) [reply templates/python/README.mustache:10]: We could also just write a static link to the repo, as this won't change

File: templates/python/README.mustache [hunk 2]
```diff
@@ -3,58 +3,25 @@
...
-### Tests
-
-Execute `pytest` to run the tests.
-
 ## Getting Started
 
-Please follow the [installation procedure](#installation--usage) and then run the following:
-
-{{> common_README }}
+[Examples](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}/tree/main/examples) for the usage of the package can be found in the [github repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```
Comment (joaopalet) [inline templates/python/README.mustache:27]:
```suggestion
[Examples](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}/tree/main/examples) for the usage of the package can be found in the [GitHub repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```

File: templates/python/README.mustache (hunk 1 above)
Comment (MelvinKl) [reply templates/python/README.mustache]: We could. But I'm not sure what we would gain from this change. It is currently producing the link, and the generator is more general, so why change it to a static link?

**Review by joaopalet — APPROVED** [review]

**Pull request merged**